### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by quentinus95**

### CODE REVIEW

**Comment by quentinus95** [comment]
ping @PLoic can you have a look ? Thx

File: README.md [hunk 1]
```diff
@@ -1,3 +1,74 @@
...
+### Image creation
+
+Build the image using the following command :
+
+```sh
+docker build -t documentation-image .
+```
+
+Once the image is built, you can run the documentation in development or production
+mode using one of the following aa :
```
Comment (PLoic) [inline README.md]: Remove "aa"

**Review by PLoic — CHANGES_REQUESTED** [review]

**Comment by quentinus95** [comment]
@PLoic I have updated the Readme with information about Docker Hub, can you review it ? Thx

**Comment by PLoic** [comment]
@quentinus95 Look good to me :)

**Comment by quentinus95** [comment]
thx

**Pull request merged**